lib/msf/core/exploit/remote/http_client.rb

```diff
@@ -150,30 +150,8 @@ def connect(opts={})
 
     client_username = opts['username'] || datastore['HttpUsername'] || ''
     client_password = opts['password'] || datastore['HttpPassword'] || ''
-
-    if datastore['HttpTrace']
-      proc_httptrace = proc { |request, response| 
-        request_color, response_color = 
-          (datastore['HttpTraceColors'] || 'red/blu').split('/').map { |color| "%bld%#{color}" }
-        
-        request = request.to_s(headers_only: datastore['HttpTraceHeadersOnly'])
-        print_line('#' * 20)
-        print_line('# Request:')
-        print_line('#' * 20)
-        print_line("%clr#{request_color}#{request}%clr")
-
-        print_line('#' * 20)
-        print_line('# Response:')
-        print_line('#' * 20)
-        
-        if response
-          response = response.to_terminal_output(headers_only: datastore['HttpTraceHeadersOnly'])
-          print_line("%clr#{response_color}#{response}%clr")
-        else
-          print_line('No response received')
-        end
-      }
-    end
+    
+    http_trace_proc = set_http_trace_proc(datastore['HttpTrace'], datastore['HttpTraceHeadersOnly'], datastore['HttpTraceColors'])
```

Comment:
```suggestion
    http_trace_proc = set_http_trace_proc(http_trace, http_trace_headers_only, http_trace_colors)
```
You are already defining these accessor methods within your class yet you aren't using them here and you are using them in the other library? Seems a bit odd :) This will also make this line shorter and easier to read.